dev-java/jcodings/jcodings-1.0.55.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# Skeleton command:
+# java-ebuilder --generate-ebuild --workdir . --pom pom.xml --download-uri https://github.com/jruby/jcodings/archive/refs/tags/jcodings-1.0.55.tar.gz --slot 1 --keywords "~amd64 ~x86" --ebuild jcodings-1.0.55.ebuild
+
+EAPI=7
+
+JAVA_PKG_IUSE="doc source test"
+MAVEN_ID="org.jruby.jcodings:jcodings:1.0.55"
+JAVA_TESTING_FRAMEWORKS="junit-4"
+
+inherit java-pkg-2 java-pkg-simple
+
+DESCRIPTION="Byte based encoding support library for java"
+HOMEPAGE="https://github.com/jruby/jcodings"
+SRC_URI="https://github.com/jruby/${PN}/archive/refs/tags/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="1"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="
+	>=virtual/jdk-11:*
+"
+
+RDEPEND="
+	>=virtual/jre-11:*
```

Comment:
@vaukai this is something new to introduce to you. this package does not compile with source 1.8 because it uses modules and those were first introduced in java 9. but still, the library can be successfully compiled for java 1.8 (that is in bytecode for java 1.8). in other words, the library needs java 11 for compilation but it can run using java 1.8, so the apps that use it are not restricted only to java 11 and newer. so `DEPEND` will stay as it is, but `RDEPEND` should be lowered to `>=virtual/jre-1.8:*`.

Reply:
Understand.  Will you please adjust it for `jcodings` and `joni`, so I need not touch them again, please

Reply:
@vaukai anyway, it does not work in this case as `joni` wants to import module from `jcodings` and if compiled with 1.8 bytecode, it does not contain module information and hence compilation of `joni` fails. so java 11 is correct in this case.